filter_code.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="./movies.js"></script>
+    <script>
+        function year (item, index) {
+            return item.year == document.getElementById("mov").value;
+            
+        }
+
+        function myFunction () {
+            var x = movies.filter(year);
+            
+            var y = [];
```

Comment:
You don't need to create a new array. `x` is already the new array that only has movies that meet the condition. So you directly use `x` instead of copying it.